src/handbook/sales/leads.md

```diff
@@ -0,0 +1,47 @@
+### Lead status
```

Comment:
I think you're missing frontmatter:

```suggestion
---
navTitle: Lead Status
---

### Lead status
```

But also; should this page be called "prospect.md"? "Lead" is overloaded as the stage before MQL